docs/release-notes/next-ceres/announcement.md

```diff
@@ -0,0 +1,77 @@
+Ceres is the 5th protocol upgrade for aeternity; following the initial protocol Roma, we have then seen Minerva, Fortuna, Lima, and Iris. Ceres is the first protocol upgrade not part of the initial update schedule and it contains mainly smaller improvements and enhancements.
+
+# Changes summarized
+Micro-block gas limit calculation changed: Gas limit was computed from a transaction's max gas value, now it is computed from the actually used gas.
+
+* Contracts can be called by name: Just like it is possible to spend to an AENS name, it is now possible to call a function for a name that points to a contract.
+
+    * Generalized Accounts:
```

Comment:
I don't think this is supposed to be indented.

Reply:
I did a very quick dump of the confluence text.
Feel free to fix ;)

Reply:
Will do